apps/src/music/views/PackDialog.module.scss

```diff
@@ -0,0 +1,102 @@
+@import 'color.scss';
+
+// A translucent overlay that covers the entire container.
+%full-container-overlay {
+  background-color: rgba(0 0 0 / 0.5);
+  color: $neutral_dark;
+  position: absolute;
+  width: 100%;
+  height: 100%;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  top: 0;
+  left: 0;
+  z-index: 80;
+}
+
+%dialog {
+  background-color: $neutral_light;
+  border-radius: 5px;
+  padding: 20px 50px;
+  width: 80%;
+  max-height: 80%;
+}
+
+.focusLock {
+  position: absolute;
+  width: 100%;
+  height: 100%;
+}
+
+.dialogContainer {
+  @extend %full-container-overlay;
+
+  .packDialog {
+    @extend %dialog;
+    display: flex;
+    flex-direction: column;
+    position: relative;
+    background-color: $neutral_dark;
+    color: $neutral_light;
+    border: solid 1px $neutral_dark40;
+
+    .heading {
+      color: $neutral_light;
+    }
+
+    .packs {
+      width: 100%;
+      height: 100%;
+      overflow-y: scroll;
```

Comment:
Minor: can we change this to `auto`? I noticed that on my machine, I was getting the scrollbar even if there was only one row of items

<img width="1510" alt="Screenshot 2024-04-02 at 11 05 29 AM" src="https://github.com/code-dot-org/code-dot-org/assets/85528507/8b3379ea-f379-4360-a798-2c7e141fa873">

Reply:
Good catch.